lib/shortcode/html5video.php

```diff
@@ -0,0 +1,127 @@
+                ),
+                array(
+                    'label' => __( 'Video URL - OGV', 'pbsandwich' ),
+                    'attr' => 'videoogv',
+                    'type' => 'attachment',
+					'description' => __( 'Choose the OGV video to be embedded.', 'pbsandwich' ),
+                ),
+                array(
+                    'label' => __( 'Video URL - MP4', 'pbsandwich' ),
+                    'attr' => 'videomp4',
+                    'type' => 'attachment',
+					'description' => __( 'Choose the MP4 video to be embedded.', 'pbsandwich' ),
+                ),
+                array(
+                    'label' => __( 'Video Billboard', 'pbsandwich' ),
+                    'attr'  => 'poster',
+                    'type'  => 'attachment',
+					'description' => __( "Choose an image that will serve as the video's poster while the video has not been played.", 'pbsandwich' ),
+                ),
+                array(
+                    'label' => __( 'Video Width', 'pbsandwich' ),
+                    'attr' => 'width',
+                    'type' => 'text',
+					'value' => '100%',
+					'description' => __( 'Enter the width of the video, in pixels or percentage.', 'pbsandwich' ),
+                ),
+                array(
+                    'label' => __( 'Video Height', 'pbsandwich' ),
+                    'attr' => 'height',
+                    'type' => 'text',
+					'value' => '282',
+					'description' => __( 'Enter the height of the video, in pixels or percentage.', 'pbsandwich' ),
+                ),
+                array(
+                    'label' => __( 'Autoplay Video', 'pbsandwich' ),
+                    'attr' => 'autoplay',
+                    'type' => 'checkbox',
+					'description' => __( "Check this to let videos play right after they load.", 'pbsandwich' ),
+					'value' => "true",
+                ),
+                array(
+                    'label' => __( 'Preload Video', 'pbsandwich' ),
+                    'attr' => 'preload',
+                    'type' => 'checkbox',
+					'description' => __( "Check this to let videos buffer while the page loads.", 'pbsandwich' ),
+					'value' => "auto",
+                ),
+                array(
+                    'label' => __( 'Hide video controls', 'pbsandwich' ),
+                    'attr' => 'controls',
+                    'type' => 'checkbox',
+					'description' => __( "Check this to disable control of video playback.", 'pbsandwich' ),
+					'value' => 'disabled',
+                ),
+			),
+        )
+    );
+	
+}
+
+
+/**
+ * Render our shortcode
+ */
+function sandwich_html5video_shortcode( $attr, $content ) {
+		
+	$attr = wp_parse_args( $attr, array(
+		'videowebm' => '',
+		'videoogv' => '',
+		'videomp4' => '',
+        'poster' => '',
+		'autoplay' => '',
+        'preload' => '',
+		'controls' => 'disabled',
+    ) );
+	
+	$attr['videowebm'] = wp_get_attachment_url( $attr['videowebm'] );
```

Comment:
This will error if the attribute is not filled up. Check if `empty( $attr['videowebm'] )`